en/SetupView/Joystick.md

```diff
@@ -50,6 +50,25 @@ The expo slider allows you to make the sticks less sensitive in the center, allo
 
 The higher the Expo value, the flatter the curve is at the center, and steeper it is at the edges.
 
+## Advanced Settings
+
+![Joystick setup - Expo Curve](../../assets/setup/joystick_advanced.jpg)
+
+The advanced settings are usually not recommended for everyday users, as they can cause unpredicted results if changed wrongly. The following settings are available:
+
+- **Joystick Mode**: Changes what the joystick actually controls, and the mavlink messages sent to the vehicle.
+    - **Normal**: User controls as if using a regular RC radio, mavlink [MANUAL_CONTROL](https://mavlink.io/en/messages/common.html#MANUAL_CONTROL) messages are used.
+    - **Attitude**: User controls the vehicle attitude, mavlink [SET_ATTITUDE_TARGET](https://mavlink.io/en/messages/common.html#SET_ATTITUDE_TARGET) messages are used.
+    - **Position**: User controls the vehicle position, mavlink [SET_POSITION_TARGET_LOCAL_NED](https://mavlink.io/en/messages/common.html#SET_POSITION_TARGET_LOCAL_NED) messages with bitmask for **position** only are used.
+    - **Force**: User controls the forces applied to the vehicle, mavlink [SET_POSITION_TARGET_LOCAL_NED](https://mavlink.io/en/messages/common.html#SET_POSITION_TARGET_LOCAL_NED) messages with bitmask for **force** only are used.
+    - **Velocity**: User controls the forces applied to the vehicle, mavlink [SET_POSITION_TARGET_LOCAL_NED](https://mavlink.io/en/messages/common.html#SET_POSITION_TARGET_LOCAL_NED) messages with bitmask for **velocity** only are used.
+
+- **Message Frequency**: This handles how often the joystick messages are sent. The default is 25 Hz, automatically lowered to 5Hz when there are no changes.
```

Comment:
When the joystick is idle (inputs are not changing), the joystick commands are sent to the vehicle at 5Hz. When the joystick is in use (input values are changing), the joystick commands are sent to the vehicle at the (higher) frequency configured by this setting. The default is 25Hz.